.ci/Jenkinsfile

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'linux && immutable' }
+  environment {
+    REPO = 'ctags-langserver'
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    YARN_GPG = 'no'
+  }
+  options {
+    timeout(time: 1, unit: 'HOURS')
+    buildDiscarder(logRotator(numToKeepStr: '30', artifactNumToKeepStr: '20'))
+    timestamps()
+    ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+  }
+  parameters {
+    booleanParam(name: 'publish_to_npm', defaultValue: false, description: 'Publish to NPM.')
```

Comment:
I don't see any references to the publish to npm, is this required or still in progress?

Reply:
Besides, I rather prefer capital cases for the params to easily identify them, but that's my preference :)

Reply:
The NPM publish is not yet there because we are still struggling with the vault secrets. Once there, I'll add it

Reply:
BTW, I added the stage, including modifying the case of the input parameter

thanks!